src/main/java/com/three_tech_solutions/slot_app/services/interfaces/UserService.java

```diff
@@ -1,10 +1,15 @@
 package com.three_tech_solutions.slot_app.services.interfaces;
 
+import com.three_tech_solutions.slot_app.data.models.Student;
 import com.three_tech_solutions.slot_app.data.models.User;
 import org.springframework.security.core.userdetails.UserDetailsService;
 import org.springframework.security.core.userdetails.UsernameNotFoundException;
 
+import java.util.List;
+import java.util.UUID;
+
 public interface UserService extends UserDetailsService {
     @Override
     User loadUserByUsername(String username) throws UsernameNotFoundException;
+    List<Student> getUserStudents (UUID userId);
```

Comment:
Los parentesis de un metodo siempre deberian estar pegados al nombre del metodo

```suggestion
    List<Student> getUserStudents(UUID userId);
```